docs/pages/versions/unversioned/sdk/device.md

```diff
@@ -0,0 +1,244 @@
+
+  **iOS warning**: The list of the model name is maintained by the community and could miss latest devices, so it is better to use `modelId` since it unique identifies new iOS devices and are always up-to-date. We accept new pull request that helps to update the device modelName list.
+
+  Gets the device model.
+
+  - iOS: e.g., `"iPhone XS Max"`
+  - Android: e.g., `"Pixel 2"`
+
+- `Device.osName: string`
+
+  Gets the device OS name.
+
+  - iOS: e.g., `"iOS"`
+  - Android: e.g., `"Android"`
+
+- `Device.modelId: string` (iOS only)
+
+  Gets the device's mobile device codes types (machine id).
+
+  - iOS: e.g., `"iPhone7,2"`
+
+- `Device.totalMemory: number`
+
+  Gets the device's total memory, in bytes. It means the total memory accessible by the kernel. This is basically the RAM size of the device, not including below-kernel fixed allocations like DMA buffers, RAM for the baseband CPU, etc.
+
+- `Device.isDevice: boolean`
+
+  `true` if the app is running on a device, `false` if running in a simulator or emulator.
+
+- `Device.deviceType: string`
+
+  Returns the device's type as a `Device.deviceType` enum.
+
+- `Device.supportedCPUArchitectures: string[]`
+
+  Returns a list of supported processor architecture versions.
+
+  **Examples**
+
+  ```js
+  Device.supportedCPUArchitectures; // [ "arm64 v8", "Intel x86-64h Haswell", "arm64-v8a", "armeabi-v7a", "armeabi" ]
+  ```
+
+- `Device.designName: string` (Android only)
+
+  Gets the specific configuration or revision of the industrial design.
+
+  - Android: e.g., `"kminilte"`
+
+- `Device.systemBuildId: string` (Android only)
+
+  Gets the build ID string meant for displaying to the user
+
+  - Android: e.g., `"PSR1.180720.075"`
+
+- `Device.productName: string` (Android only)
+
+  Gets the device's overall product name. Chosen by the device implementer containing the development name or code name of the device.
+
+  - Android: e.g., `"kminiltexx"`
+
+- `Device.platformApiLevel: number` (Android only)
+
+  Gets the SDK version of the software currently running on this hardware device. This value never changes while a device is booted, but it may increase when the hardware manufacturer provides an OTA update. Click [here](https://developer.android.com/reference/android/os/Build.VERSION_CODES.html) to see all possible version codes and corresponding versions.
+
+  - Android: e.g., `19`
+
+- `Device.osVersion: string`
+
+  Gets the user-visible os version number.
+
+  - iOS: e.g., `12.3.1`
+  - Android: e.g., `4.0.3`
+
+- `Device.deviceName: string`
+
+  A human-readable name for the device type.
+
+  - eg., `"Vivian's iPhone XS"`
+
+- `Device.osBuildFingerprint: string` (Android only)
+
+  A string that uniquely identifies this build. It must follow the template:
+
+  $(BRAND)/$(PRODUCT)/$(DEVICE)/$(BOARD):$(VERSION.RELEASE)/$(ID)/$(VERSION.INCREMENTAL):$(TYPE)/\$(TAGS)
+
+  - `"google/sdk_gphone_x86/generic_x86:9/PSR1.180720.075/5124027:user/release-keys"`
+
+- `Device.osBuildId: string`
```

Comment:
Let's differentiate this more from the other build ID:
```suggestion
- `Device.osInternalBuildId: string`
```

Also, move this below `osBuildId` so they are grouped.